shell/browser/api/electron_api_base_window.cc

```diff
@@ -1118,7 +1118,7 @@ void BaseWindow::ResetBrowserViews() {
       // reset if the owner window is *this* window.
       if (browser_view->web_contents()) {
         auto* owner_window = browser_view->web_contents()->owner_window();
-        if (owner_window == window_.get()) {
+        if (owner_window && owner_window == window_.get()) {
```

Comment:
could we instead check `if (!window_.get()) return` at the top of the function?

Reply:
@nornagon this wouldn't fix the issue - the owner window could still be nullptr in that case